doc/appendix-de-kasssichv/appendix-de-kasssichv.md

```diff
@@ -94,12 +94,92 @@ The ftReceiptCase indicates the receipt type and defines how it should be proces
+| `0x4445000000000001` | "cash payment in national currency"                                                                                               | 0-                  |
+| `0x4445000000000002` | "cash payment in foreign currency"                                                                                                | 0-                  |
+| `0x4445000000000003` | "crossed cheque"                                                                                                                  | 0-                  |
+| `0x4445000000000004` | "debit card payment"                                                                                                              | 0-                  |
+| `0x4445000000000005` | "credit card payment"                                                                                                             | 0-                  |
+| `0x4445000000000006` | "voucher payment (coupon)"                                                                                                        | 0-                  |
+| `0x4445000000000007` | "online payment"                                                                                                                  | 0-                  |
+| `0x4445000000000008` | "customer card payment"                                                                                                           | 0-                  |
+| `0x4445000000000009` | "other debit card"                                                                                                                | 0-                  |
+| `0x444500000000000A` | "other credit card"                                                                                                               | 0-                  |
+| `0x444500000000000B` | "account receivable"<br />delivery note/ settlement in foreign currency                                                           | 0-                  |
+| `0x444500000000000C` | "SEPA transfer"                                                                                                                   | 0-                  |
+| `0x444500000000000D` | "other transfer"                                                                                                                  | 0-                  |
+| `0x444500000000000E` | "cash book expense"                                                                                                               | 0-                  |
+| `0x444500000000000F` | "cash book contribution"                                                                                                          | 0-                  |
+| `0x4445000000000010` | "levy"<br />DE: Anzahlung                                                                                                         | 0-                  |
```

Comment:
rename to down payment, this will sync naming along chargeitemcase

Reply:
which if these 4?

Reply:
0x4445000000000010

Reply:
renamed